client/src/App.tsx

```diff
@@ -1,30 +1,34 @@
 import React, { useEffect, useState } from "react";
 import { BrowserRouter } from "react-router-dom";
 import { Routes, Route } from "react-router";
-import Home from "./pages/Home";
-import SignUp from "./pages/SignUp/SignUp";
-import AdvancedSearch from "./pages/AdvancedSearch/AdvancedSearch";
+import { Home } from "./pages/Home";
+import { SignUp } from "./pages/SignUp/SignUp";
+import { AdvancedSearch } from "./pages/AdvancedSearch/AdvancedSearch";
 import { Result } from "./pages/SearchResult/SearchResult";
-import NotFound from "./pages/NotFound/NotFound";
-import Login from "./pages/Login/Login";
-import Review from "./pages/Review/ReviewsList/SingleReview";
-import ReviewsListParent from "./pages/Review/ReviewsList/ReviewsListParent";
+import { NotFound } from "./pages/NotFound/NotFound";
+import { Login } from "./pages/Login/Login";
+import { SingleReview } from "./pages/Review/ReviewsList/SingleReview";
+import { ReviewListParent } from "./pages/Review/ReviewsList/ReviewsListParent";
 
-import GlobalStyle from "./theme/globalStyle";
+import { GlobalStyle } from "./theme/globalStyle";
 import { Navbar } from "./components/NavBar/NavBar";
 import { SingleSearchResultPage } from "./pages/SearchResult/SingleSearchResult";
 import "./App.css";
+import { apiServer } from "./utils/helpers";
 
-const App = () => {
+export default function App() {
   const [currentUserEmail, setCurrentUserEmail] = useState("");
+  const [currentUser, setCurrentUser] = useState("");
   useEffect(() => {
     (async () => {
-      const response = await fetch("/api/user/user", {
+      const response = await fetch(`${apiServer()}/api/user/get-user`, {
+        method: "POST",
         headers: { "Content-Type": "application/json" },
         credentials: "include",
       });
       const content = await response.json();
-      setCurrentUserEmail(content.username);
+      setCurrentUserEmail(content.email);
+      setCurrentUser(content);
```

Comment:
does this need to be content.username?

Reply:
Do we want to have users login with a username or email? I can set it up either way, you choose

Reply:
I remember now, I used email because thats how David setup the login page, but we can change that. You let me know how you want it setup and I will take care of it.

Reply:
Okay I see what you are saying where Login has email field, we should keep it as that then. Future development would be great for the field to accept either email or username to login.